tdrs-backend/tdpservice/settings/common.py

```diff
@@ -281,10 +281,12 @@ class Common(Configuration):
     )
 
     # Sessions
-    SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+    SESSION_ENGINE = "tdpservice.core.custom_session_engine"
+    SIGNED_COOKIE_EXPIRES = 60 * 60 * 24  # 24 hours
     SESSION_COOKIE_HTTPONLY = True
-    SESSION_EXPIRE_AT_BROWSER_CLOSE = True
```

Comment:
The setting causes the session expiry to be set to browser close and ignore SESSION_COOKIE_AGE.